test/passing/attributes.ml

```diff
@@ -215,17 +215,19 @@ let _ = ([] @ []) [@a]
 let _ =
   f
     (( a_____________________________________
-     @ b_____________________________________ ) [@a])
+     @ b_____________________________________ )
+     [@a])
 
 (* Attribute should wrap as a block *)
 let _ =
   ( a_______________________________________________________________________
-  @ b________________________________________________________________ ) [@a]
+  @ b________________________________________________________________ )
+  [@a]
```

Comment:
Looks like this attribute should fit. If not, this test isn't checking the property it was intended for anymore and should be modified.

Reply:
I think this test still wants either modified or removed. I introduced it as part of https://github.com/ocaml-ppx/ocamlformat/pull/1306 to guard against a bug where the attribute was split across multiple lines on wrap.

Reply:
Ok. I don't have a strong position either way. @gpetiot feel free to include a change to this test in this PR as you see fit.